File: C,C++/baseball/baseball_14/Guess.cpp
#include "Constant.h"
#include <iostream>
#include "Guess.h"
using namespace std;

void Guess::input()
{
    for (int i = 0; i < DIGIT; i++)
        std::cin >> numbers[i];
}

const char* Guess::getPrefix()
{
    return "[Guess] ";
}

File: C#/baseball/Guess.cs
using System;

namespace BaseballCSharp
{
    public class Guess : NumberContainer
    {
        public void Input()
        {
            for (int i = 0; i < _numbers.Length; i++)
            {
                string line = Console.ReadLine();
                _numbers[i] = int.Parse(line);
            }
        }

        protected override string GetPrefix()
        {
            return "[추측] ";
        }
    }
}